resources/js/bots/wallet/bot.js

```diff
@@ -352,11 +352,26 @@ function handleSend(params, context) {
         gasPrice: calculateGasPrice(params["bot-db"]["sliderValue"])
     };
 
-    try {
-        return web3.eth.sendTransaction(data);
-    } catch (err) {
-        return {error: err.message};
-    }
+
+    web3.eth.sendTransaction(data, function(error, hash) {
+        if (error) {
+            status.sendSignal("handler-data", {
+                status: "failed",
+                message_id: context["message-id"],
+                error: error
```

Comment:
we could send
```js
{
 'message-id': context["message-id"], 
 'handler-data': 
 {
    status: "failed",
    error: error
 }
}
```
We already have `message-id` in message object, no need to store this data again

Reply:
@rasom I'm not sure I follow, do you object against sending `message_id` in data object (second argument to `status.sendSignal`) ? 
How will I know on the receiving side which message-id was originally passed in the context map to `handleSend` call without explicitly passing it back in `status.sendSignal` ?